CRM/Import/Parser.php

```diff
@@ -14,7 +14,7 @@
  * @package CRM
  * @copyright CiviCRM LLC https://civicrm.org/licensing
  */
-abstract class CRM_Import_Parser {
+class CRM_Import_Parser {
```

Comment:
I'm not sure I see the reason for removing `abstract`. Is there a reason this class would be used directly?

Reply:
@colemanw - yeah it's only that I removed the last abstract funciton - but I can put it back - hopefully at some point we will come up with a better interface & migrate the classes over...

Reply:
Ok. I'd say let's keep it `abstract` for now.

Reply:
@colemanw ok - I also removed the change to the Custom partser class so this won't conflict with https://github.com/civicrm/civicrm-core/pull/23222 (I'll just have to follow up once both are merged)